README.md

```diff
@@ -119,10 +125,35 @@ docker run -it --rm \
   -v ~/.vagrant.d:/.vagrant.d \
   -v $(pwd):$(pwd) \
   -w $(pwd) \
+  --network host \
   vagrantlibvirt/vagrant-libvirt:latest \
     vagrant status
 ```
 
+It's possible to define an alias in `~/.bashrc`, for example:
+```bash
+alias vagrant='
+  mkdir -p ~/.vagrant.d/{boxes,data,tmp}; \
+  docker run -it --rm \
+    -e LIBVIRT_DEFAULT_URI \
+    -v /var/run/libvirt/:/var/run/libvirt/ \
+    -v ~/.vagrant.d:/.vagrant.d \
+    -v $(pwd):$(pwd) \
+    -w $(pwd) \
+    --network host \
+    vagrantlibvirt/vagrant-libvirt:latest \
+    vagrant'
+```
+
+Known issue of Docker based Installation: SSH command option cannot be used, so below command:
+```bash
+vagrant ssh master -c 'ls -la'
+```
```

Comment:
I wasn't aware of this, guessing bad shell expansion on the entrypoint script part

Reply:
I understand, it's a rare case. Fortunately, there is a simple workaround.

Reply:
#1197 resolved it for me